pkg/apis/openebs/v1alpha1/devicerequest_types.go

```diff
@@ -2,21 +2,51 @@ package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
 )
 
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
 // DeviceRequestSpec defines the desired state of DeviceRequest
 type DeviceRequestSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	Capacity  uint64 `json:"capacity"`  // disk size in bytes
+	DriveType string `json:"driveType"` // DriveType represents the type of drive like SSD, HDD etc.,
```

Comment:
should this be `devicetype` instead of drive type?

Reply:
done